serviceregistration/kubernetes/client/config.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	"crypto/x509"
+	"io/ioutil"
+	"net"
+	"os"
+
+	"github.com/hashicorp/vault/sdk/helper/certutil"
+)
+
+const (
+	// This is Kubernetes' default namespace.
+	DefaultNamespace = "default"
+
+	// These environment variables aren't set by default.
+	// Vault may read them in if set through these environment variables.
+	// Example here:
+	// https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+	EnvVarKubernetesNamespace = "NAMESPACE"
```

Comment:
Wondering if this should be `VAULT_NAMESPACE` and `VAULT_POD_NAME` since these are pretty generic names?

Reply:
Oh good idea! Will do!